### ISSUE TITLE
Disabling compression by default

### ISSUE THREAD

**Issue by multiformeingegno** [pivpn/pivpn#613]
OpenVPN Connect for iOS the official app just released a new version 3.0.2(894) that disables compression by default for reported security reasons. Compression is enabled by default in a clean PiVPN install - is it worth disabling?

More info https://forums.openvpn.net/viewtopic.php?f=36&t=27186

**Comment by Giraffe1966** [comment]
PR #597 addresses this issue. It has not been merged yet.

**Comment by PnjDbq** [comment]
Not sure if this is a bug or working as intended, but my connection on iOS 12.0.1 ONLY works with compression on.  There does not appear to be a difference between "full" and "downlink only".  I'm encountering some kind of routing/DNS resolution error when compression is set to "no".

The default change in the iOS app v3.0.2 broke my VPN until I figured out that compression is required.

Question is:  Is requiring compression by design?  Or should the server work without compression and I have a different issue to diagnose..?

Server: OpenVPN 2.4.5
iOS: 12.0.1
App: 3.0.2(894)

**Comment by Giraffe1966** [comment]
As per the OpenVPN man page (which can be found [here](https://community.openvpn.net/openvpn/wiki/Openvpn24ManPage)), to disable compression, leave the `--compress` option empty to disable compression.

**Comment by PnjDbq** [comment]
Figured out what I'm doing and might as well document it for others.

Based on #617 and #623, it appears that the server, .ovpn profile, and iOS app need to all be in agreement in regards to the compression setting.  When the iOS default changed, one of those settings was not like the other two.

...I'm actually not sure why the iOS app lets the user specify compression since all three must be in agreement, but that's for another day.  FWIW, the Windows client program does not appear to have a compression setting and uses what the profile specifies.

In any case, I was able comment out the compression flag on the server by editing `/etc/openvpn/server.conf` on the Pi and and restarting the server.  

The `.ovpn` profiles had to be edited to match.  Not an issue on the desktop, but was not sure how to edit on iOS, so I regenerated with `pivpn -a`.  The script included the compression flag, but I was able to edit the profile on the server before copying to iOS.
